.github/CODEOWNERS

```
@@ -1,23 +1,12 @@
 *                   @ethereum-optimism/security-reviewers
 
-/runbooks/**/*mcp-l1* @sbvegan @ethereum-optimism/security-reviewers
+/runbooks/**/*mcp-l1* @ethereum-optimism/devrel @ethereum-optimism/security-reviewers
```

My understanding is there are plans to add an MCP to FP runbook that will also be used by devrel, so perhaps we relax this to cover all runbooks
```suggestion
/runbooks @ethereum-optimism/devrel @ethereum-optimism/security-reviewers
```